src/button.cpp

```diff
@@ -19,7 +19,7 @@ bool Button::read( QTextStream &stream ) {
 //	at this point, toDefault() has just been called.
 
     //read in a line of text and break it into words
-    QString input = stream.readLine().toLower();
+    QString input = stream.readLine();
```

Comment:
Why did you change this here?
And instead of allocating so many temporary strings with every `toLower()` down below why not use `QString::compare(str1, str2, Qt::CaseInsensitive)`?

Reply:
The issue here was that setting the entire string to lowercase caused issues with layout handling - since layout names are case sensitive.

Would this be better solved by a separate `inputLower` variable, or should I use the string comparison method you mentioned?
I do apologize for the amount of comparisons, C++ is still a bit new to me so I didn't consider it an issue.

Reply:
Yeah, I thing using `QString::compare(str1, str2, Qt::CaseInsensitive)` would probably be best.

Reply:
Implemented :grin: